packages/common/src/wizard/wizard.d.ts

```diff
@@ -1,5 +1,6 @@
 import { AnyObject, Field } from "@data-driven-forms/react-form-renderer";
 
+export const CONDITIONAL_SUBMIT_FLAG = '@@ddf-common-wizard__conditional-submit-step';
```

Comment:
maybe it could be in a separate file so people do not have to import everything else in rare cases

Reply:
If you use this constant without the common wizard, you would need to implement the whole wizard logic therefore this constant no longer serve any purpose.

Reply:
Yeah, but you can have just a schema where you want to use this value...

Or you can maybe implement your own wizard component but keep the constant the same.

Reply:
Ill split it